src/components/editor/Bindings/Selector.tsx

```diff
@@ -1,21 +1,62 @@
+import { Clear } from '@mui/icons-material';
 import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline';
-import { ListItemText } from '@mui/material';
+import { Box, IconButton, ListItemText, TypographyProps } from '@mui/material';
 import {
     DataGrid,
     GridColDef,
     GridRenderCellParams,
     GridSelectionModel,
     GridValueGetterParams,
 } from '@mui/x-data-grid';
+import CollectionPicker from 'components/collection/Picker';
 import SelectorEmpty from 'components/editor/Bindings/SelectorEmpty';
+import { alternativeDataGridHeader, slateOutline } from 'context/Theme';
 import { useEffect, useRef, useState } from 'react';
 import { useUnmount } from 'react-use';
+import { useFormStateStore_isActive } from 'stores/FormState';
 import {
     useResourceConfig_currentCollection,
+    useResourceConfig_removeCollection,
     useResourceConfig_resourceConfig,
     useResourceConfig_setCurrentCollection,
+    useResourceConfig_setRestrictedDiscoveredCollections,
 } from 'stores/ResourceConfig';
 
+interface BindingSelectorProps {
+    readOnly?: boolean;
+}
+
+interface DeleteButtonProps {
+    collection: string;
+    disabled: boolean;
+}
+
+function DeleteButton({ collection, disabled }: DeleteButtonProps) {
+    const removeCollection = useResourceConfig_removeCollection();
+
+    const setRestrictedDiscoveredCollections =
+        useResourceConfig_setRestrictedDiscoveredCollections();
+
+    const handlers = {
+        removeCollection: (event: React.MouseEvent<HTMLElement>) => {
+            event.preventDefault();
```

Comment:
So this is kind of an issue... the prevent default isn't stopping anything. The row click action still fires. This means that delete is working via a weird side effect.

If you add `event.stopPropagation` then it stops the rowClick and delete no longer works correctly. I do not think this is a blocking issue for the PR but this will need to get fixed later.